src/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/ReflectionExecutionDomainCallbacksImplementation.cs

```diff
@@ -31,9 +31,19 @@ public ReflectionExecutionDomainCallbacksImplementation(ExecutionDomain executio
             _executionEnvironment = executionEnvironment;
         }
 
-        public sealed override Type GetType(string typeName, Func<AssemblyName, Assembly> assemblyResolver, Func<Assembly, string, bool, Type> typeResolver, bool throwOnError, bool ignoreCase)
+        public sealed override Type GetType(string typeName, Func<AssemblyName, Assembly> assemblyResolver, Func<Assembly, string, bool, Type> typeResolver, bool throwOnError, bool ignoreCase, string defaultAssemblyName)
```

Comment:
Not something to fix right now. String won't work once CoreRT supports AssemblyLoadContexts and can load different copies of the same assembly. Maybe add TODO comment about it somewhere.